packages/rest/src/keys.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: @loopback/rest
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {CoreBindings} from '@loopback/core';
+
+export namespace RestBindings {
+  // RestServer-specific bindings
+  export const CONFIG = `components.RestServer.config`;
```

Comment:
Since the config is read from application config by default, is it a good idea to keep a copy in a new binding?
 I'd prefer `RestBindings.CONFIG` to say something like `CoreBindings.APPLICATION_CONFIG + '#rest'`.

Reply:
Deferring until #587 is complete.